### PR TITLE
Add Preact integration

### LINKED ISSUE #18: Preact integration

### PR DESCRIPTION
**Pull request by xadamy**
Closes #18.

### CODE REVIEW

File: packages/preact/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+```
+
+## Usage
+
+### Error Boundary
+
+If you are using React v16 or higher, you can use the `ErrorBoundary` component to catch errors from anywhere in the child component tree.
+
+```jsx
+import React from "react"
```
Comment (tombruijn) [inline packages/preact/README.md]: react and preact have the same imports?

File: packages/preact/README.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  <ErrorBoundary 
+    instance={appsignal} 
+    tags={{ tag: "value" }} {/* Optional */}
+    fallback={(error) => <FallbackComponent />} {/* Optional */}
+  >
+    { /** Child components here **/}
+  </ErrorBoundary>
+)
+
+ReactDOM.render(<App />, document.getElementById("root"))
```
Comment (tombruijn) [inline packages/preact/README.md]: I see Preact is recommending the `render` function on it's own? https://preactjs.com/guide/switching-to-preact#4-simplify-root-render
should we also do that in our docs?

File: packages/preact/README.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+```
+yarn add @appsignal/javascript @appsignal/preact
+npm install --save @appsignal/javascript @appsignal/preact
+```
+
+## Usage
+
+### Error Boundary
+
+If you are using React v16 or higher, you can use the `ErrorBoundary` component to catch errors from anywhere in the child component tree.
```
Comment (tombruijn) [inline packages/preact/README.md]: it doesn't work on a lower version? Should we explicitly mention that?

File: packages/preact/README.md (hunk 1 above)
Comment (xadamy) [reply packages/preact/README.md]: Fixed in 17c5cbf.

File: packages/preact/README.md (hunk 2 above)
Comment (xadamy) [reply packages/preact/README.md]: Fixed in 17c5cbf.

**Comment by xadamy** [comment]
Thanks @tombruijn, opened an issue for this here: https://github.com/appsignal/appsignal-examples/issues/5

**Pull request merged**